lib/mayaUsd/ufe/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ if(CMAKE_UFE_V2_FEATURES_AVAILABLE)
             UsdTransform3dSetObjectMatrix.cpp
             UsdTransform3dUndoableCommands.cpp
             UsdUIInfoHandler.cpp
+            UsdUIUfeObserver.cpp
```

Comment:
I put both files in the V2 block since they don't rely on anything new from UFE.